tests/core/cgo/asm_conditional_cgo/asm_cgo.go

```diff
@@ -0,0 +1,16 @@
+//go:build amd64 || arm64
```

Comment:
Slightly surprised that this works without the `cgo` constraint here. In the non-cgo case, isn't `callASM` defined twice?